### PR TITLE
Adding `tensorly-quantum` and `tensorly-torch` from PyPI

### PR DESCRIPTION
**Pull request by sugatoray**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged. `NA`
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by sugatoray** [comment]
## Reason for Build Failure

Trying to run `import tltorch`, fails with `ModuleNotFoundError`. This was also tested on Colab and the same error occurs there as well. The issue has been reported to the maintainers of `tensorly-torch`: https://github.com/tensorly/torch/issues/13.

[(build error)](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=431501&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=1248)

```sh
import: 'tltorch'
Traceback (most recent call last):
  File "/home/conda/staged-recipes/build_artifacts/tensorly-torch_1640110985920/test_tmp/run_test.py", line 2, in <module>
    import tltorch
  File "/home/conda/staged-recipes/build_artifacts/tensorly-torch_1640110985920/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/python3.9/site-packages/tltorch/__init__.py", line 4, in <module>
    from . import factorized_tensors
  File "/home/conda/staged-recipes/build_artifacts/tensorly-torch_1640110985920/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/python3.9/site-packages/tltorch/factorized_tensors/__init__.py", line 1, in <module>
    from .factorized_tensors import (CPTensor, TuckerTensor, TTTensor,
  File "/home/conda/staged-recipes/build_artifacts/tensorly-torch_1640110985920/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/python3.9/site-packages/tltorch/factorized_tensors/factorized_tensors.py", line 8, in <module>
    import tensorly as tl
ModuleNotFoundError: No module named 'tensorly'
Tests failed for tensorly-torch-0.3.0-pyhd8ed1ab_0.tar.bz2 - moving package to /home/conda/staged-recipes/build_artifacts/broken
WARNING:conda_build.build:Tests failed for tensorly-torch-0.3.0-pyhd8ed1ab_0.tar.bz2 - moving package to /home/conda/staged-recipes/build_artifacts/broken
WARNING conda_build.build:tests_failed(2970): Tests failed for tensorly-torch-0.3.0-pyhd8ed1ab_0.tar.bz2 - moving package to /home/conda/staged-recipes/build_artifacts/broken
TESTS FAILED: tensorly-torch-0.3.0-pyhd8ed1ab_0.tar.bz2
```

---

### Error on Colab

colab-error

File: recipes/tensorly-torch/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
+    - nose
+    - numpy    
+    - scipy
+    - pytorch >=1.5.1
```
Comment (jaimergp) [inline recipes/tensorly-torch/meta.yaml:27]: I think you need `tensorly` here too:

```suggestion
    - pytorch >=1.5.1
    - tensorly
```

**Comment by sugatoray** [comment]
@jaimergp Thank you. It worked.

**Comment by sugatoray** [comment]
The windows build fails as PyTorch is not available on conda-forge for windows.

**Comment by sugatoray** [comment]
@jaimergp Thanks for merging.

**Pull request merged**